tests/by-util/test_kill.rs

```diff
@@ -82,6 +82,29 @@ fn test_kill_list_all_signals_as_table() {
         .stdout_contains("HUP");
 }
 
+#[test]
+fn test_kill_table_starts_at_1() {
+    new_ucmd!()
+        .arg("-t")
+        .succeeds()
+        .stdout_matches(&Regex::new("^\\s?1").unwrap());
```

Comment:
I would also check that it is the `HUP` signal, see also my other comment.

Reply:
`HUP` signal check has been added to regex.